mis_builder/models/kpimatrix.py

```diff
@@ -546,12 +546,12 @@ def as_dict(self):
     # semantic domain occur.
 
     @classmethod
-    def _make_row_id(cls, kpi_id: int, account_id: int | None) -> str:
+    def _make_row_id(cls, kpi_id: int, account_id: int) -> str:
```

Comment:
Could you do like this?

```suggestion
    def _make_row_id(cls, kpi_id: int, account_id: Optional[int]) -> str:
```

with `from typing import Optional` at the top.

Reply:
ruff doesn't allow that: https://github.com/OCA/mis-builder/actions/runs/15213332995/job/42792688070?pr=684#step:7:51

I can however use that as last straw to upgrade my customer's python version, as it will be EOL anyways and we forget about 3.9 compatibility

Reply:
As you prefer. Else just remove the type annotations.

Reply:
reverted to the initial version. Or did you mean remove all annotations?

Reply:
Wait, could you try with `from __future__ import annotations` at the top of the file, and keep `int | None` ?

That should work with 3.9 too.

Reply:
ah right, better